www/src/utils/typography.js

```diff
@@ -299,7 +292,7 @@ const _options = {
           )}`,
         },
       },
-      [MIN_LARGER_DISPLAY_MEDIA_QUERY]: {
+      [presets.VVHd]: {
```

Comment:
Ha, is this _very_ _very_ HD?

Reply:
👍LOL yeah… :/ I'll open an issue around

> That said, I'm planning on revisiting those breakpoint names and values in a bit…;-)

now capturing the renaming of those.

Reply:
It's actually kinda funny and not all that unclear ;) Pattern I've seen most often is usually `xx`, e.g. `xsmall`, `small`, `medium`, `large`, `xlarge`, `xxlarge`, but I think both work relatively well here 🙃

Reply:
#8270